python/doc/user_manual/base_objects.rst

```diff
@@ -89,14 +89,10 @@ Vectors
     :toctree: _generated/
     :template: class.rst_t
 
-    BoolCollection
     Description
     Indices
-    IndicesCollection
     Point
     PointWithDescription
-    ComplexCollection
-    ScalarCollection
```

Comment:
I think Regis had a Point: Collection classes must be documented even if we should avoid to promote them in test/examples as they may still get used, so I suggest that you drop the commits that remove Collection_doc.i.in and reinstate Collection objects here.
Sorry for letting you believe that they could be dropped all the way.

Reply:
Ok. Fixed.